pyproject.toml

```diff
@@ -55,7 +55,6 @@ conda = "conda.cli.main_pip:main"
 source = "vcs"
 
 [tool.hatch.version.raw-options]
-version_scheme = "calver-by-date"
```

Comment:
by specifying the `calver-by-date` version scheme hatch is unable to derive the next development/canary version from a `YY.MM.0` version, because `0` is not a valid date, see the workflow failure linked above

this change has no impact on our production releases, those still use the tagged version (where we are manually conforming to our `YY.MM.N` scheme)

AFAIK falling back to `guess-next-dev` is what we want for canaries